config/packages/security.yaml

```diff
@@ -29,9 +29,13 @@ security:
     # Easy way to control access for large sections of your site
     # Note: Only the *first* access control that matches will be used
     access_control:
-        - { path: ^/admin, roles: ROLE_ADMIN }
+        - { path: ^/admin, roles: ROLE_API_USER }
         # - { path: ^/profile, roles: ROLE_USER }
 
+    role_hierarchy:
```

Comment:
Can this be backed be enums (to remove typing errors)?

https://symfony.com/blog/new-in-symfony-6-2-improved-enum-support

Reply:
Changed